src/test/java/org/jenkinsci/plugins/workflow/cps/PersistenceProblemsTest.java

```diff
@@ -244,7 +243,6 @@ public void inProgressNormal() throws Exception {
     }
 
     @Test
-    @Ignore
```

Comment:
These tests have been ignored since they were added. I am not sure why, but I went ahead and unignored them.

Reply:
You probably already knew this, but I just noticed that these test cases seem to be duplicated in `workflow-job-plugin`'s [`CpsPersistenceTest`](https://github.com/jenkinsci/workflow-job-plugin/blob/f369d5bdfc39ac902f544a125e971414cb51a463/src/test/java/org/jenkinsci/plugins/workflow/job/CpsPersistenceTest.java). Once the dust settles, it might be worth doing a sweep of the two copies to ensure that they are in sync with each other.

Reply:
Yeah, I think I will probably delete the copies in workflow-job after this. Ideally we would run the PCT in the CI builds here with just the core Pipeline plugins so that changes are tested against the newest versions of everything.